modules/auth/site/templates/block_account_info.tpl

```diff
@@ -0,0 +1,60 @@
+ 	<div class="primary-tabs">
+		{if isset="$user"}
+		<h3>Account Information</h3>
+		<div>
+			<div>
+				<div>	
+			   		<label>Username:</label>  {$user.name/}
+			   	</div>
+			   	<div>	
+			   		<label>Email:</label>  {$user.email/}
+			   	</div>
+			   	<div>	
+			   		<label>Creation Date:</label>  {$user.creation_date/}
+			   	</div>
+			   	<div>	
+			   		<label>Last login:</label>  {$user.last_login_date/}
+			   	</div>	
+			   	<div>
+			   		<form method="get" action="{$site_url/}account/{$strategy/}">
+    					<button type="submit">Logout</button>
```

Comment:
ok I read a bit on that ... button or input ... are the same for this case, but button are easier to style.
so it is recommended to use  button type="submit"  as you did.
Glad I learnt something on that topic today :)